<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PXDMFReaderPanel</class>
 <widget class="QWidget" name="PXDMFReaderPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="Refresh">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>103</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Point Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="PointArrayStatus" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         
         <property name="iconSize">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Cell Data</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="CellArrayStatus" native="true">
         <property name="tabKeyNavigation" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">  
    <widget class="QTabWidget" name="tabWidget2">
    <widget class="QWidget" name="tab_21">
     <attribute name="title">
       <string>Visualization Space</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_20">
      <property name="margin">
        <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTreeWidget" name="VisualizationSpaceStatus">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute> 
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>103</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alternatingRowColors">
          <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
          <bool>false</bool>
        </property>
        <property name="itemsExpandable">
          <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_22">
      <attribute name="title">
       <string>Visualization Time</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_21">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="VisualizationTimeStatus">
          <attribute name="headerVisible">
            <bool>false</bool>
          </attribute>   
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>103</height>
            </size>
          </property>
          <property name="maximumSize">
            <size>
              <width>16777215</width>
              <height>16777215</height>
          </size>
        </property>
          <property name="alternatingRowColors">
            <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
            <bool>false</bool>
          </property>
          <property name="itemsExpandable">
            <bool>false</bool>
          </property>
          <property name="sortingEnabled">
            <bool>false</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
            <bool>false</bool>
          </attribute>
          <column>
            <property name="text">
              <string notr="true">Visualization Time</string>
            </property>
          </column>
        </widget>
      </item>
      </layout>
    </widget> 
   </widget> 
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="horizontalLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="UseInterpolation">
       <property name="text">
        <string>Use Interpolation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="ComputeDerivatives">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Compute Derivatives</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="ContinuousUpdate">
       <property name="text">
        <string>Continuous Update</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Detach">
       <property name="text">
        <string>Detach Fixed Dimensions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
